### PR TITLE
Add PR #1689

### PR DESCRIPTION
**Pull request by ariard**

### CODE REVIEW

File: _posts/2022-09-13-#1689.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+layout: pr
+date: 2022-09-13
+title: "Expose new BumpChannelClose event for channels with anchor outputs"
+authors: ["wpaulino"]
+pr: 1689
+components: ["tests"]
```
Comment (dunxen) [inline _posts/2022-09-13-#1689.md:7]: I think we just need to change this from “tests” to the module/area of code we’re working in. I think this was left from one of the other copy pasta for the other one. Oops! 😅

File: _posts/2022-09-13-#1689.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+host: ariard
+status: future
```
Comment (dunxen) [inline _posts/2022-09-13-#1689.md:9]: This should be `upcoming` instead of future for the template logic to work.

Comment (ariard) [reply _posts/2022-09-13-#1689.md:9]: Yep, updated to `upcoming`.

File: _posts/2022-09-13-#1689.md (hunk 1 above)
Comment (ariard) [reply _posts/2022-09-13-#1689.md:7]: Yeah sure, changed to "chain monitoring", though let me know if better suggestion!

**Comment by dunxen** [comment]
Oh I don't think the changes actually came through with your last push 😅

**Comment by ariard** [comment]
Done at 6ca4823

> Oh I don't think the changes actually came through with your last push sweat_smile

Classic me!

**Review by dunxen — APPROVED** [review]

**Pull request merged**